config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  resources :articles
-  root 'welcome#show'
+  resources :articles do
+    resources :comments
+  end
+  root 'articles#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.